go/private/rules/wrappers.bzl

```diff
@@ -53,20 +53,24 @@ def go_binary_macro(name, srcs=None, cgo=False, cdeps=[], copts=[], clinkopts=[]
       **kwargs
   )
 
-def go_test_macro(name, srcs=None, cgo=False, cdeps=[], copts=[], clinkopts=[], **kwargs):
+def go_test_macro(name, srcs=None, deps=None, importpath="", library=None, embed=[], gc_goopts=[], cgo=False, cdeps=[], copts=[], clinkopts=[], **kwargs):
   """See go/core.rst#go_test for full documentation."""
-  cgo_info = None
-  if cgo:
-    cgo_info = setup_cgo_library(
-        name = name,
-        srcs = srcs,
-        cdeps = cdeps,
-        copts = copts,
-        clinkopts = clinkopts,
-    )
+  library_name = name + "~library~"
+  go_library(
+      name = library_name,
+      visibility = ["//visibility:private"],
+      srcs=srcs,
```

Comment:
Nit: add space around = for all keyword arguments (here and in `go_test` below).